doc/parameters.md

```diff
@@ -402,6 +402,15 @@ Path to a file with the X.509 certificate secret key in the PEM format.
 Defines the maximum length of the client certificate chain.
 If this parameter has `1` as its value, it is possible to include an additional certificate in the client certificate chain. For example, root certificate authority.
 
+### `APICAST_HTTPS_VERIFY_CLIENT`
+
+**Default:** `optional_no_ca`
+**Values:**
+- `off`: Don't request client certificates and don't do client certificate verification.
```

Comment:
```suggestion
- `off`: Do not request client certificates or perform client certificate verification.
```

Reply:
Suggesting the removal of the contraction and use of "don't" twice in one line.

Reply:
Fixed.